docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -15,7 +15,43 @@ Ansible is an agentless automation tool that you install on a single host (refer
 Control node requirements
 =========================
 
-For your control node (the machine that runs Ansible), you can use nearly any UNIX-like machine with Python 3.8 or newer installed. This includes Red Hat, Debian, Ubuntu, macOS, BSDs, and Windows under a `Windows Subsystem for Linux (WSL) distribution <https://docs.microsoft.com/en-us/windows/wsl/about>`_. Windows without WSL is not natively supported as a control node; see `Matt Davis' blog post <http://blog.rolpdog.com/2020/03/why-no-ansible-controller-for-windows.html>`_ for more information.
+For your _control_ node (the machine that runs Ansible), you can use nearly any UNIX-like machine with Python 3.8 or newer installed. This includes Red Hat, Debian, Ubuntu, macOS, BSDs, and Windows under a `Windows Subsystem for Linux (WSL) distribution <https://docs.microsoft.com/en-us/windows/wsl/about>`_. Windows without WSL is not natively supported as a control node; see `Matt Davis' blog post <http://blog.rolpdog.com/2020/03/why-no-ansible-controller-for-windows.html>`_ for more information.
+
+.. _managed_node_requirements:
+
+Managed node requirements
+=========================
+
+The _managed_ node (the machine that Ansible is managing) does not require Ansible to be installed, but requires Python 2.7, or Python 3.5 - 3.11 to run Ansible library code.
```

Comment:
Network devices can be managed with Ansible even though they do not have Python installed - can we add a link here to the network docs (see https://docs.ansible.com/ansible/latest/network/getting_started/network_differences.html#execution-on-the-control-node)? and/or a note saying "Ansible can also manage some devices that cannot run Python. For example, many network devices do not have Python installed."?

Reply:
Technically to manage a network device, the target becomes localhost from a module execution perspective.  So it still applies in the end.  It's just a caveat of a different execution mechanism performed for network devices.